ui/src/shared/components/TimeSeries.tsx

```diff
@@ -344,6 +349,7 @@ const mstp = (state: AppState, props: OwnProps): StateProps => {
 
 const mdtp: DispatchProps = {
   notify: notifyAction,
+  setQueryResults: setQueryResultsByQueryID,
```

Comment:
what's the idea behind aliasing this? it's really confusing - there's already a method being exported from another file called `setQueryResults`.  this really confounds the ability to see where `setQueryResults` is used - this represents a false positive of that.

Reply:
Fair point. I think aliasing helps differentiate between the imported action vs the one that's accessible in the props. Without aliasing, I find that my workflow is:

1. Import the action
2. Try and call the action directly within the component
3. Get confused for a bit, do some debugging, then realize the err of my ways

Reply:
for `differentiate between the imported action vs the one that's accessible in the props` we sometimes use `setX` vs `onSetX` I had personally set a bad example of this in this file.